src/SFA.DAS.EmployerIncentives.Web/Services/LegalEntities/LegalEntitiesService.cs

```diff
@@ -30,7 +30,7 @@ public async Task<IEnumerable<LegalEntityModel>> Get(string accountId)
 
             response.EnsureSuccessStatusCode();
 
-            var data = await JsonSerializer.DeserializeAsync<IEnumerable<LegalEntityDto>>(await response.Content.ReadAsStreamAsync());
+            var data = await JsonSerializer.DeserializeAsync<IEnumerable<LegalEntityDto>>(await response.Content.ReadAsStreamAsync(), options: new JsonSerializerOptions { PropertyNameCaseInsensitive = true });
```

Comment:
Is it worthwhile writing this as extension method otherwise we have to remember to write it each time?

Reply:
Or making it case insensitive, but...

Reply:
actually I added this json setting to my controllers in the outer api, not sure if that affected reading from external sources

Reply:
services.AddControllers(c=>
                {
                    if (!_env.IsLocalOrDev())
                    {
                        c.Filters.Add(new AuthorizeFilter("default"));
                    }
                })
                .AddJsonOptions(options => {
                    options.JsonSerializerOptions.PropertyNameCaseInsensitive = true;
                });

Reply:
Unfortunately it's a static so not possible.  Others are grumbling about it.

https://github.com/dotnet/runtime/issues/31094

Reply:
Could make it a service. ISerialiser.Deserialize<T>( ... )